tox.ini

```diff
@@ -7,7 +7,7 @@
 envlist = py27, flake8
 
 [pytest]
-addopts = --cov=mozetl tests/
+addopts = --cov=mozetl
```

Comment:
Why the change here? Do you think that might be related to the version conflict failure?

Reply:
This change is specifically for being able to run parallel tests. The filenames need to be passed directly to pytest, but this additional option is causing pytest to scan through this entire directory which is a super-set of the tests in the files. 

I remember running into issues when I didn't specify the test directory, but it looks like pytest does the right thing when run in the root directory.